src/views/open-api-docs-view/utils/get-response-data.ts

```diff
@@ -0,0 +1,72 @@
+// Utils
+import { getPropertyDetailPropsFromSchemaObject } from './get-property-detail-props'
+// Types
+import type { PropertyDetailsProps } from '../components/property-details'
+import type { OpenAPIV3 } from 'openapi-types'
+
+/**
+ * Map known response codes to human-readable headings.
+ *
+ * If an response code isn't in this map, we use the response code itself
+ * as the heading.
+ */
+const RESPONSE_CODE_HEADINGS: Record<string, string> = {
+	'200': 'Successful Response',
+	default: 'Default Response',
```

Comment:
Not sure if something like this would be helpful https://github.com/prettymuchbryce/http-status-codes

Reply:
Nice, yeah that's super helpful! Have updated to show `Default Response` if the OpenAPI response code is `default` (which isn't a supported code in `http-status-codes`), and otherwise grab `getReasonPhrase(responseCode)` using `http-status-codes` and combine that with the code itself for a more informative heading 👍